plugins/lsp/default.nix

```diff
@@ -192,12 +192,13 @@ in
       keymapsOnEvents.LspAttach =
         let
           mkMaps =
-            prefix:
+            prefix: descPrefix:
             mapAttrsToList (
               key: action:
               let
                 actionStr = if isString action then action else action.action;
-                actionProps = if isString action then { } else filterAttrs (n: v: n != "action") action;
+                actionProps = optionalAttrs (!(isString action)) (filterAttrs (n: v: n != "action") action);
```

Comment:
This seems unrelated?

Especially if https://github.com/nix-community/nixvim/pull/2595/files#r1865065311 is done, we wouldn't need _any_ changes to this let-in block for this PR.

I don't mind having general cleanup if it's in a separate commit or another PR. If that was done, perhaps `actionStr` could be simplified to `action.action or action`? But both changes really feel unrelated to this PR :slightly_smiling_face:

Reply:
#2598